src/include/miopen/gemm_v2.hpp

```diff
@@ -38,8 +38,6 @@ enum GemmBackend_t
 {
     nogemmbackend = 0,
     rocblas       = 1,
-    miopengemm    = 2,
-    miopentensile = 3,
```

Comment:
This struct is redundant now!

Reply:
@JehandadKhan maybe not, please see https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2265#issuecomment-1695963969